sdcm/logcollector.py

```diff
@@ -630,6 +630,8 @@ def collect_logs_per_node(node):
 
         final_archive = self.archive_dir_with_zip64(self.local_dir)
         s3_link = self.upload_logs(final_archive, "{0.test_id}/{0.current_run}".format(self))
+        remove_files(self.local_dir)
+        remove_files(final_archive)
```

Comment:
why do you delete `final_archive` if it stored inside the `self.local_dir`

Reply:
It is not stored in self.log_dir, it is outside, the zip contains self.local_dir

Reply:
self.local dir stores the raw and archive files for cluster type, and then it zipped to "self.local_dir.zip"

Reply:
I didn't say it is stored in log_dir...

Reply:
@bentsi sorry, mistype. updated my comments.